sql/core/src/main/scala/org/apache/spark/sql/execution/datasources/v2/WriteToDataSourceV2Exec.scala

```diff
@@ -259,27 +266,25 @@ case class AppendDataExec(
  */
 case class OverwriteByExpressionExec(
     table: SupportsWrite,
+    writeBuilder: WriteBuilder,
```

Comment:
If the builder is no longer passed in, then I don't think this class needs to change at all.